supervisor/worker.go

```diff
@@ -65,6 +65,13 @@ func (w *worker) Start() {
 		}
 		if err := process.Start(); err != nil {
 			logrus.WithField("error", err).Error("containerd: start init process")
+			t.Err <- err
```

Comment:
on the exit error the call to `container.Delete()` will also kill the init process if it exists